modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/source/SourceWithParameterGroupExclusiveOptionalsOneRequiredTestCase.java

```diff
@@ -83,6 +77,56 @@ public void testWithComplexParameterDslTrue() throws Exception {
                is("hello bird!"));
   }
 
+  @Test
+  public void testShowInDslTrueWithRepeatedNameParameter() throws Exception {
+    startFlow("dslTrueRepeatedNameParameter");
+    assertEventsFired();
+    SomeParameterGroupOneRequiredConfig config = getParameterGroupConfigValue();
+    assertThat(config.getRepeatedNameParameter(), is("hello cat!"));
+  }
+
+  @Test
+  public void testShowInDslTrueWithComplexParameterWithRepeatedNameParameter() throws Exception {
+    startFlow("dslTrueComplexParameterWithRepeatedNameParameter");
+    assertEventsFired();
+    SomeParameterGroupOneRequiredConfig config = getParameterGroupConfigValue();
+    Assert.assertThat(config.getComplexParameter().getRepeatedNameParameter(), CoreMatchers.is("hi bird!"));
```

Comment:
replace CoreMatches.is with static import

Reply:
replaced